components/pipeline-service/development/main-pipeline-service-configuration.yaml

```diff
@@ -1684,6 +1684,7 @@ spec:
       kube-api-qps: 50
       replicas: 2
       threads-per-controller: 32
+      statefulset-ordinals: true
```

Comment:
e2e tests are failing waiting for pipeline service to be "synced" from ArgoCD stand point.
```
Waiting 10 seconds for application sync
pipeline-service-in-cluster-local                  OutOfSync   Healthy
```